pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -121,11 +121,13 @@ type CommandInitOption struct {
 	CRDs                               string
 	ExternalIP                         string
 	ExternalDNS                        string
+	PullSecrets                        []string
 	CertValidity                       time.Duration
 	KubeClientSet                      kubernetes.Interface
 	CertAndKeyFileData                 map[string][]byte
 	RestConfig                         *rest.Config
 	KarmadaAPIServerIP                 []net.IP
+	imagePullSecrets                   []corev1.LocalObjectReference
```

Comment:
Sorry I didn't notice this variable.

I think we don't need the `imagePullSecrets` as it's essentially an `internal variable` of `getImagePullSecrets`.